commit e61bc57a395dbc16551cd3dfb887012a0c6025ff (refs/changes/35/935/3)
Author: June Yi <june.yi@samsung.com>
Date:   2016-11-27T22:23:33+09:00 (2 years, 10 months ago)
    
    joyent/sdcadm#28 --force-data-path is not needed for `sdcadm create portolan`

diff --git a/lib/cli/do_create.js b/lib/cli/do_create.js
index 8fa8909..4ca0c52 100644
--- a/lib/cli/do_create.js
+++ b/lib/cli/do_create.js
@@ -89,7 +89,8 @@ function do_create(subcmd, opts, args, cb) {
                 changes: changes,
                 progress: self.progress,
                 uuid: self.uuid,
-                skipHACheck: opts.skip_ha_check
+                skipHACheck: opts.skip_ha_check,
+                forceDataPath: true
             }, function (err, plan_) {
                 plan = plan_;
                 next(err);
